<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ClientWindow</class>
 <widget class="QMainWindow" name="ClientWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>多协议客户端 v1.0</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string>服务器配置</string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>服务器IP：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="serverIpEdit">
         <property name="text">
          <string>127.0.0.1</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>TCP端口：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="tcpPortEdit">
         <property name="text">
          <string>8888</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>WS端口：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="wsPortEdit">
         <property name="text">
          <string>9090</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <widget class="QWidget" name="tcpTab">
       <attribute name="title">
        <string>TCP文件传输</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QPushButton" name="tcpConnectBtn">
          <property name="text">
           <string>连接TCP服务器</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLineEdit" name="filePathEdit">
            <property name="placeholderText">
             <string>选中的文件路径</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="selectFileBtn">
            <property name="text">
             <string>选择文件</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="sendFileBtn">
            <property name="text">
             <string>发送文件</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="httpTab">
       <attribute name="title">
        <string>HTTP业务请求</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QPushButton" name="getBooksBtn">
            <property name="text">
             <string>图书浏览</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="getUserBtn">
            <property name="text">
             <string>用户管理</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="getCartBtn">
            <property name="text">
             <string>购物车</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="getOrderBtn">
            <property name="text">
             <string>订单</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="getCollectBtn">
            <property name="text">
             <string>收藏</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QTextEdit" name="httpResponseEdit">
          <property name="readOnly">
           <bool>true</bool>
          </property>
          <property name="placeholderText">
           <string>HTTP响应内容将显示在这里</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="wsTab">
       <attribute name="title">
        <string>WebSocket实时通信</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QPushButton" name="wsConnectBtn">
          <property name="text">
           <string>连接WebSocket服务器</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTextEdit" name="wsMsgEdit">
          <property name="readOnly">
           <bool>true</bool>
          </property>
          <property name="placeholderText">
           <string>消息记录将显示在这里</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QLineEdit" name="wsMsgInput">
            <property name="placeholderText">
             <string>输入要发送的消息</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="sendWsMsgBtn">
            <property name="text">
             <string>发送消息</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="title">
       <string>操作日志</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <widget class="QTextEdit" name="logEdit">
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>900</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
